### PR TITLE
Added Kuttl test for installation of  kyverno

### PR DESCRIPTION
**Pull request by nsathyaseelan**
Added Kuttl test for Installation/uninstallation of Kyverno. This test contains below steps 

- Installation of kyverno
- Verify the installed kyverno components
- Install and verify the kyverno policies (Running the test cases  present in the kyverno-policies repo)
- Uninstall kyverno charts
- Verify the Kyverno related components are removed

### CODE REVIEW

File: .github/workflows/e2e.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  run-e2etest:
+    strategy:
+      fail-fast: false
+
+      matrix:
+        k8s-version: [v1.25.2, v1.22.1]
```
Comment (anusha94) [inline .github/workflows/e2e.yaml:21]: use latest patch in a given minor series

File: test/e2e/t01-n4k-installation/01-install-kyverno.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    helm repo add nirmata https://nirmata.github.io/kyverno-charts
+    helm repo update nirmata
+    helm install kyverno nirmata/kyverno -n kyverno --create-namespace --set licenseManager.licenseKey=ifkbGamY65BgJzbauN2dcsx0uDZqK39kDi618sovyqhkc+A2mEmLLj8T6YifVbvY9iARbJjEno/5XyBgKlSS3IIvOcd3dt2ZCq395e4tAxXzIvbiTxKOiPCuPukx5qDUwG8VlnHWRkTytG5lYa3kcLVtxC6a3dwxodvjSD/q6R6WsMScg5VA2Gx8b5YDOR+8VPF52po3HFwITqXDMm7f6FXv1k+d4dXM7ShM9btcEV4SFAsqcIWwEcpCleeaOEPaFQjHXs1pcfiIGdlHMXPSDvRaIo/OwBJPjnEH8jejfF4K0fcxxoJ51/78OUAlvy8pIF9O4cuyi4UGIi3B9wSTqg== --version=$N4K_VERSION
```
Comment (anusha94) [inline test/e2e/t01-n4k-installation/01-install-kyverno.yaml:7]: put this in a GH secret - we will raise PRs from a branch and not fork

File: test/e2e/t01-n4k-installation/04-kyverno-policy-test.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    cd ../../../kyverno-policies
```
Comment (anusha94) [inline test/e2e/t01-n4k-installation/04-kyverno-policy-test.yaml:5]: where is this dir?

File: test/e2e/t01-n4k-installation/01-install-kyverno.yaml (hunk 2 above)
Comment (anusha94) [inline test/e2e/t01-n4k-installation/01-install-kyverno.yaml:7]: let's run with 3 replicas

File: test/e2e/t01-n4k-installation/04-kyverno-policy-test.yaml (hunk 3 above)
Comment (nsathyaseelan) [reply test/e2e/t01-n4k-installation/04-kyverno-policy-test.yaml:5]: we were cloning the repo in the github workflow.

**Pull request merged**